test/helper.rb

```diff
@@ -2,6 +2,7 @@
 require 'webmock/test_unit'
 require 'rubygems/mock_gem_ui'
 require 'json/jwt'
+require 'byebug'
```

Comment:
making debugging in tests a bit easier so we don't have to type `require 'byebug';`